src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -304,6 +332,10 @@ class ContentExplorer extends Component {
             newSelectedItems[item.id] = item;
         }
 
+        if (onSelectedItemsUpdate) {
+            onSelectedItemsUpdate(newSelectedItems);
+        }
```

Comment:
any particular reason this is before `setState`? if not, could we move it after since that's the pattern for `onFoldersPathUpdate`?